release-notes/docs/3.12.0.md

```diff
@@ -0,0 +1,117 @@
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+<<< TODO >>>
+
+## Upgrade notes
+
+- This upgrade can be rolled out remotely. Users will download the new version in the background and be prompted to reload the app when it's ready. A small amount of data will be needed to download the new version.
+- Upgrading to this version does not require other applications to be upgraded. This release does not drop support for any hardware or software that works with the previous version.
+- This release modifies some CouchDB view definitions, so we recommend you Stage the upgrade and wait for the views to be rebuilt before rolling it out to minimize downtime.
+- It is recommended to test the upgrade on a clone of your server first to ensure it works well for your application.
+
+### Breaking changes
+
+None.
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://docs.communityhealthtoolkit.org/core/overview/supported-software/) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+### UI/UX changes
+
+None.
+
+## Highlights
+
+### Telemetry
+
+The CHT-Core is collecting new telemetry data:
+
+- [#6300](https://github.com/medic/cht-core/issues/6300): The difference between the client datetime and the server datetime, it helps to detect user's clock errors.
+- [#6915](https://github.com/medic/cht-core/issues/6915): Access to granular telemetry data for post-hoc analysis, monitoring & research initiatives.
+- [#6354](https://github.com/medic/cht-core/issues/6354): Access to replication information.
+
+### Monitoring
+
+The Monitoring API version 1 (`/api/v1/monitoring`) is now deprecated and the version 2 (`/api/v2/monitoring`) is introduced with new metrics available:
+
+- [#6607](https://github.com/medic/cht-core/issues/6607): The number of users that have connected to the api in a given number of days, it helps to detect when users are unable to connect to the server.
+- [#6572](https://github.com/medic/cht-core/issues/6572): Outgoing messages information, it helps to determine the SMS failures rate.
+
+### Contact summary, targets and tasks
+
+The contact summary, targets and tasks now have more data available and a new CHT API with useful functions: 
+
+- [#6914](https://github.com/medic/cht-core/issues/6914): Ability to verify user's permissions from contact summary, targets and tasks by using the CHT API.
+- [#6919](https://github.com/medic/cht-core/issues/6919): Access to UHC "home visits" information from contact summary.
+
+### SMS
+
+The CHT-Core will send a SMS when the origin phone number is not registered in the app:
+
+- [#6650](https://github.com/medic/cht-core/issues/6650): Send an error message to users when `sys.facility_not_found` is generated.
+
+## And more...
```

Comment:
I reckon we should have content under this heading.  If we do not have anything, then I guess it is better we do niot have it.

Reply:
This is autogenerated by the script but I think the `## And more...` is to introduce the following sections (as they have smaller title ### ) `### Features, ### Improvements`.  🤔